feat: BROS-3: Generate only technical info for tag docs

hlomzik opened the pull request:
Tag docs are getting more complex, so better to handle all text and illustrations directly in `.md` files and auto-generate only technical info:
- tag parameters table
- fields of result produced by control tags
- sample result json

Split doc files into `.md` files basically keeping them in place and snippets with autogenerated info, stored in new `docs/includes/tags` directory. Include snippets into main doc files with `insertmd` directive:

```
{% insertmd includes/tags/paragraphs.md %}
```

Workflow is partially reverted from #7446

hlomzik committed:
feat: BROS-3: Generate only technical info for tag docs

Parameters table, result props and sample result json — that's it.
Put them into new `docs/includes/tags` directory.

Workflow is partially reverted from #7446

hlomzik committed:
Try to remove all excess data from `create-docs` script

hlomzik committed:
Fix path for includes

robot-ci-heartex committed:
ci: Build tag docs

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15113122804

hlomzik committed:
Test includes on TimeSeries

hlomzik committed:
Remove excess headers

hlomzik committed:
Simplify create-docs script

Remove supertags, is_new param, fetching old docs.
Add more comments.
Remove commented out code.

hlomzik committed:
Use `insertmd` directive to use generated details

robot-ci-heartex committed:
ci: Build tag docs

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15115492413

caitlinwheeless reviewed (approved):
I think "Result parameters" works well! ❤️

hlomzik commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/15121178361)
> Successfully merged:  create mode 100644 web/libs/ui/src/lib/sparkles/utils.ts

hlomzik committed:
Merge branch 'develop' into 'fb-bros-3/tag-docs'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15121178361

robot-ci-heartex committed:
ci: Build tag docs

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15121221389

hlomzik committed:
Update pdf.md to use `insertmd`

hlomzik committed:
Fix linting
